fdbserver/DataDistribution.actor.h

```diff
@@ -59,10 +59,12 @@ struct IDataDistributionTeam {
 	virtual bool isWrongConfiguration() = 0;
 	virtual void setWrongConfiguration(bool) = 0;
 	virtual void addServers(const vector<UID> &servers) = 0;
+	virtual UID getTeamID() = 0;
 
 	std::string getDesc() {
 		const auto& servers = getLastKnownServerInterfaces();
-		std::string s = format("Size %d; ", servers.size());
+		std::string s = format("[TeamID - %s]", getTeamID().shortString());
```

Comment:
"[]"  is kind of not friendly for `grep`.
Maybe change the format to `TeamID:%s` or `TeamID %s`?

Reply:
Done